requirements/dev.txt

```diff
@@ -55,7 +55,7 @@ coverage[toml]==6.4.4
     #   -r requirements/quality.txt
     #   codecov
     #   pytest-cov
-diff-cover==6.5.1
+diff-cover==7.0.1
```

Comment:
No breaking change in the new version, only Python 3.6 support has been dropped. https://github.com/Bachmann1234/diff_cover/blob/main/CHANGELOG#L1